orcid-web/src/main/webapp/static/javascript/ng1Orcid/app/modules/emailsForm/emailsForm.component.ts

```diff
@@ -224,6 +224,11 @@ export class EmailsFormComponent implements AfterViewInit, OnDestroy, OnInit {
 
                     if (this.inputEmail.errors.length == 0) {
                         this.initInputEmail();
+                    } else {
+                        if (this.inputEmail.errors.filter(error => error.startsWith('Please enter your email address in the').length > 0)
```

Comment:
@DanielPalafox this is not going to work for non english error messages right?

Reply:
@DanielPalafox ping! have you found a different solution for this?

Reply:
@amontenegro I fixed it looking an email which is consistent in the error message in different languages